src/app/games/go/tests/GoMinimax.spec.ts

```diff
@@ -59,7 +60,7 @@ describe('GoMinimax', () => {
             const moves: GoMove[] = minimax.getListMoves(initialNode);
             expect(moves).toEqual([new GoMove(1, 1)]);
         });
-        it('should want to switch dead piece when it consider those pieces alive', () => {
+        it('should want to switch dead piece when it consider those pieces alive (for white)', () => {
```

Comment:
clear chez nous ! (player zero peut-être serait même mieux ici ?)

Reply:
clear chez nous ??
J'l'ai fait en mode go, au go, on dit comme ça !
Tu veux que j'uniformise pour MGP :D ?

Reply:
Dans le tuto on parle pourtant de clair et foncé ! Je partirai sur une uniformisation oui

Reply:
Eh c'est pas clear hein ! c'est light !
No need to ticket this my lord, if took care of it!